app/model/Tag.scala

```diff
@@ -1,47 +1,46 @@
-import com.gu.tagmanagement.{Tag => ThriftTag, TagType, TagReindexBatch}
+import com.gu.tagmanagement.{BlockingLevel => ThriftAdBlockingLevel, Tag => ThriftTag, TagType, TagReindexBatch}
 import helpers.XmlHelpers._
 import repositories.{SponsorshipRepository, SectionRepository}
 import scala.util.control.NonFatal
 import scala.xml.Node
 
 case class Tag(
-  id: Long,
-  path: String,
-  pageId: Long,
-  `type`: String,
-  internalName: String,
-  externalName: String,
-  slug: String,
-  hidden: Boolean = false,
-  legallySensitive: Boolean = false,
-  comparableValue: String,
-  categories: Set[String] = Set(),
-  section: Option[Long],
-  publication: Option[Long],
-  description: Option[String] = None,
-  parents: Set[Long] = Set(),
-  externalReferences: List[Reference] = Nil,
-  podcastMetadata: Option[PodcastMetadata] = None,
-  contributorInformation: Option[ContributorInformation] = None,
-  publicationInformation: Option[PublicationInformation] = None,
-  isMicrosite: Boolean,
-  capiSectionId: Option[String] = None,
-  trackingInformation: Option[TrackingInformation],
-  campaignInformation: Option[CampaignInformation],
-  activeSponsorships: List[Long] = Nil,
-  sponsorship: Option[Long] = None, // for paid content tags, they have an associated sponsorship but it may not be active
-  paidContentInformation: Option[PaidContentInformation] = None,
-  expired: Boolean = false,
-  var updatedAt: Long = 0L
+                id: Long,
+                path: String,
+                pageId: Long,
+                `type`: String,
+                internalName: String,
+                externalName: String,
+                slug: String,
+                hidden: Boolean = false,
+                legallySensitive: Boolean = false,
+                comparableValue: String,
+                categories: Set[String] = Set(),
+                section: Option[Long],
+                publication: Option[Long],
+                description: Option[String] = None,
+                parents: Set[Long] = Set(),
+                externalReferences: List[Reference] = Nil,
+                podcastMetadata: Option[PodcastMetadata] = None,
+                contributorInformation: Option[ContributorInformation] = None,
+                publicationInformation: Option[PublicationInformation] = None,
+                isMicrosite: Boolean,
+                capiSectionId: Option[String] = None,
+                trackingInformation: Option[TrackingInformation],
+                campaignInformation: Option[CampaignInformation],
+                activeSponsorships: List[Long] = Nil,
+                sponsorship: Option[Long] = None, // for paid content tags, they have an associated sponsorship but it may not be active
+                paidContentInformation: Option[PaidContentInformation] = None,
+                expired: Boolean = false,
+                adBlockingLevel: Option[BlockingLevel],
+                contributionBlockingLevel: Option[BlockingLevel],
```

Comment:
Do `adBlockingLevel` needs to be an Option[] ? or maybe with BlockingLevel.None as default value ?
Just a thought